app/src/main/res/values/strings.xml

```diff
@@ -16,5 +16,11 @@
 	<string name="fragment_request_search">Search</string>
     <string name="fragment_request_post_id">Post id</string>
     <string name="fragment_request_invalid_input">Invalid input, please enter a valid id.</string>
+    <!-- Login -->
+    <string name="fragment_email">Email</string>
+    <string name="fragment_password">Password</string>
+    <string name="fragment_login">Log in</string>
+    <string name="fragment_signup">Sign up</string>
+    <string name="fragment_conditions">By joining you agree with the terms and conditions.</string>
```

Comment:
Here the label is perfect. You should change your string resources names to `login_email`, `login_...`. Fragment is not a "good" choice because almost every view is a fragment :p